packages/Newtonsoft.Json.11.0.2/LICENSE.md

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
If this is required - I don't want it in the repository.
Please bring it in as a git submodule, or some other NuGet magic?